integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -64,18 +64,20 @@ def get_unreviewed_pages_context(self):
         :return: Dictionary containing the context for unreviewed pages
         :rtype: dict
         """
-        unreviewed_pages = (
+        last_versions = (
             PageTranslation.objects.filter(
-                status=status.REVIEW,
                 language__slug=self.request.region.default_language.slug,
                 page__id__in=Subquery(
                     self.request.region.non_archived_pages.values("pk")
                 ),
             )
             .order_by("page__id", "language__id", "-version")
             .distinct("page__id", "language__id")
-            .all()
         )
+        unreviewed_pages = PageTranslation.objects.filter(
+            id__in=Subquery(last_versions.values("pk")),
+            status=status.REVIEW,
+        ).all()
```

Comment:
```suggestion
        )
```
See https://github.com/digitalfabrik/integreat-cms/pull/2499#discussion_r1377257856